docker-compose.yaml

```diff
@@ -8,6 +8,7 @@ services:
       - hugo
       - server
       - -D
+      - --ignoreCache
```

Comment:
Not required apparently. Please remove again.

Reply:
Fixed.